Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -947,6 +952,11 @@ public UserMap getUserMap() {
         return userMap;
     }
 
+    @Override
+    public BalanceTopImpl getBalanceTop() {
```

Comment:
Best to return `BalanceTop` here so people using `Essentials` will import the API class rather than impl